### ISSUE TITLE
awsume -l shows region as None when the Region has in fact been set

### ISSUE THREAD

**Issue by felipe1982** [trek10inc/awsume#13]
`aws configure --profile myuser set region ap-southeast-2`

This sets the region.

`aws configure --profile myuser get region` returns `ap-southeast-2` as expected.

`awsume -l` does not show it.

```
=====================AWS Profiles=====================
PROFILE             TYPE  SOURCE  MFA?  REGION
default             User  None    No    ap-southeast-2
myuser              User  None    No    None
```

The default appears correctly, as expected.

**Comment by mbarneyjr** [comment]
I will take a look and get back to you as soon as I can, thanks for the report!

**Comment by mbarneyjr** [comment]
Alright, I've taken a look, and released an update. The issue was only with the list-profiles feature. Everything else should have been functioning normally. The issue has been resolved if you update to 1.2.7, and everything should be displayed consistent with your config and credentials files. Thanks again for the report!
